mitmproxy/proxy/layers/http/_http2.py

```diff
@@ -448,6 +452,10 @@ def handle_h2_event(self, event: h2.events.Event) -> CommandGenerator[bool]:
             self.streams[event.stream_id] = StreamState.HEADERS_RECEIVED
             yield ReceiveHttp(ResponseHeaders(event.stream_id, response, bool(event.stream_ended)))
             return False
+        elif isinstance(event, h2.events.TrailersReceived):
+            pseudo_trailers, trailers = split_pseudo_headers(event.headers)
```

Comment:
https://httpwg.org/specs/rfc7540.html#rfc.section.8.1.2.1:

> Pseudo-header fields MUST NOT appear in trailers. 

So we can skip that here :)